src/containers/pages/JurisdictionAssignment/JurisdictionTable/index.tsx

```diff
@@ -169,7 +191,7 @@ const JurisdictionTable = (props: JurisdictionSelectorTableProps) => {
   // been set, if drilling down has begun.
   let derivedParentNode = currentParentNode;
   let derivedChildrenNodes = currentChildren;
-  if (!props.currentParentId) {
+  if (!props.currentParentId && !mapCurrentParent.currentParentId.length) {
```

Comment:
Why do we need both props.currentParentId and mapCurrentParent.currentParentId?  These should be the same value at all times, no?

Also, I prefer that we destructure `currentParentId` from props